alphabase/peptide/fragment.py

```diff
@@ -777,27 +864,19 @@ def remove_unused_fragments(
         Multiple fragment dataframes can be provided which will all be sliced in the same way. 
         This allows to slice both the fragment_mz_df and fragment_intensity_df. 
         At least one fragment dataframe needs to be provided. 
-
-    frag_start_col : str, optional
-        Fragment start idx column in `precursor_df`, such as "frag_start_idx" and "peak_start_idx".
-        Defaults to "frag_start_idx".
-
-    frag_stop_col : str, optional
-        Fragment stop idx column in `precursor_df`, such as "frag_stop_idx" and "peak_stop_idx".
-        Defaults to "frag_stop_idx".
     
     Returns
     -------
     pd.DataFrame, List[pd.DataFrame]
         returns the reindexed precursor DataFrame and the sliced fragment DataFrames
     """
 
-    precursor_df = precursor_df.sort_values([frag_start_col], ascending=True)
-    frag_idx = precursor_df[[frag_start_col,frag_stop_col]].values
+    precursor_df = precursor_df.sort_values(['frag_start_idx'], ascending=True)
```

Comment:
Put `frag_start_col` and `frag_stop_col` into paramters to make this fn also work for other kinds of input df

Reply:
They are already parameters, why they are still constants here?